cmd/start.go

```diff
@@ -9,6 +12,10 @@ import (
 	"github.com/celestiaorg/celestia-node/node"
 )
 
+var (
+	errIncorrectAddress = errors.New("incorrect address provided")
+)
```

Comment:
nit: we don't make errors as a separate var unless there is one more usage for it or at least there should a plan/use-case on how it can be used later.

nit2: it would be nice if the error can report the connection to the remote core. We have multiple addresses, so it may be confusing which address the error is telling about. Appending `for Remote Core` would fix this.

Reply:
@vgonkivs

Reply:
@Wondertan , done.